package.json

```diff
@@ -20,7 +27,6 @@
     "@hapi/hapi": "18.x.x",
```

Comment:
Could/should hapi, glue, etc. be production dependencies (i.e. should this flavor be similar to the "deployment" flavor)?

Reply:
The idea was that this would be cherry-picked onto the `deployment` flavor. I wasnt quite sure how to set that up, but that is the end goal. Location of these deps is very much subject to change for whent he deployment flavor is picked and i can figure out clean applies.

Reply:
Cool, I'll think on that.